java-manta-client-unshaded/src/main/java/com/joyent/manta/exception/MantaErrorCode.java

```diff
@@ -172,6 +172,12 @@
      * Known component sources: muskie, piranha.
      */
     INVALID_AUTH_TOKEN_ERROR("InvalidAuthenticationToken"),
+    /**
+     * @since 3.4.1
```

Comment:
For this an all subsequent `@since` entries, wouldn't buckets be released in a 3.X.0 release, not a minor one?